odoo/tools/translate.py

```diff
@@ -316,6 +317,8 @@ def xml_translate(callback, value):
         result = translate_xml_node(root, callback, parse_xml, serialize_xml)
         # remove tags <div> and </div> from result
         return serialize_xml(result)[5:-6]
+    except ValueError:
```

Comment:
are we sure that **all** value errors are because of an extra `<?xml version=1.0 encoding="utf-8"?>` that is it not possible to have a value error for anything else?